applications/tari_app_grpc/proto/types.proto

```diff
@@ -228,3 +228,37 @@ message ConsensusConstants {
     /// Block weight for kernels
     uint64 block_weight_kernels = 16;
 }
+
+
+
+message Address{
+    bytes address =1;
+    string last_seen = 2;
+    uint32 connection_attempts = 3;
+    uint32 rejected_message_count = 4;
+    uint64 avg_latency = 5;
+}
+
+message Peer{
```

Comment:
Protobuf uses `//` for comments and the proto builder will make them into rust docstrings